src/demo/coupons.js

```diff
@@ -99,9 +99,9 @@ const getAllCodes = (couponHasError = false) => [...Array(codesCount)].map((_, i
 
   const maxUsed = maxRedemptionsCount - redemptionsAvailablePerCode;
   const redemptionsUsedPerCode = isAssigned ? faker.random.number({
-    min: minRedemptionsCount + 1,
-    max: codeHasError ? redemptionsAvailablePerCode : maxUsed,
-  }) : minRedemptionsCount;
+    min: minRedemptionsCount,
```

Comment:
If we're always setting the `min` to `1`, we won't have any unassigned codes with 0 redemptions used.

Reply:
Nevermind, I see that this is taken into account on [line 104](https://github.com/edx/edx-portal/pull/161/files#diff-f6af3eb2f15b8537b5eba2432574fe84R104).